spec/index.bs

```diff
@@ -2450,6 +2457,30 @@ GPUBuffer includes GPUObjectBase;
     : <dfn>usage</dfn>
     ::
         The allowed usages for this {{GPUBuffer}}.
+
+    : <dfn>state</dfn>
+    ::
+        The current <dfn enum for=>GPUBufferMapState</dfn> of the buffer:
+
+        <dl dfn-type=enum-value dfn-for=GPUBufferMapState>
+            : <dfn>"unmapped"</dfn>
+            ::
+                `this`.{{GPUBuffer/[[state]]}} is [=buffer state/unmapped=] or
```

Comment:
Ah, I see this is noted as something to consider in https://github.com/gpuweb/gpuweb/issues/3013#issuecomment-1147989788

Reply:
if we do keep them merged like this, then it would be good to come up with a better name. "unmapped" makes it sound like you can use it in a submit, which is wrong.